Add TAC Minutes & Supporting Docs For 2024-04-04 (#227)
confidential-computing/governance: TAC/Meetings/2024/2024-04-04/CCC-TAC-Minutes-2024-04-04.md

* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

#### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* Arm - Nathaniel McCallum +  / Michael Lu
* Google - Cfir Cohen  / Catalin Sandu 
* Huawei - Zhipeng (Howard) Huang 
* Intel - Dan Middleton (TAC chair) + / Simon Johnson
* Meta - Kevin Hui + / Henry Wang +
* Microsoft - Alec Fernandez +
* Nvidia - Fritz Alder +
* Red Hat - Lily Sturmann  / Yash Mankad +
* TikTok - Mingshen Sun + / Yao Zhang

   " + " indicates attending

#### Other Attendees

* Bob Reiter (Booz Allen Hamilton)
* Bokdeuk Jeong (Samsung)
* Drasko Draskovic
* Gregory Starkey (Booz Allen Hamilton)
* Henk Birkholz
* Jorg Rodel (Suse)
* Manu Foutaine (Hushmesh)
* Matthieu Legre (Cysec)
* Nathaniel McCallium (Red Hat)
* Nick Vidal
* Nicklas Maeding (IBM)
* Pawan Khandavilli (Microsoft)
* Rachel Wan (IBM)
* Richard Zak (Enarx Maintainer; Booz Allen Hamilton)

> * Tyler Fanelli (Red Hat)

Reviewer: ```suggestion
* Thomas Fossati (Linaro)
* Tyler Fanelli (Red Hat)
```

* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

#### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* Arm - Nathaniel McCallum +  / Michael Lu
* Google - Cfir Cohen  / Catalin Sandu 
* Huawei - Zhipeng (Howard) Huang 
* Intel - Dan Middleton (TAC chair) + / Simon Johnson
* Meta - Kevin Hui + / Henry Wang +
* Microsoft - Alec Fernandez +
* Nvidia - Fritz Alder +
* Red Hat - Lily Sturmann  / Yash Mankad +
* TikTok - Mingshen Sun + / Yao Zhang

   " + " indicates attending

#### Other Attendees

* Bob Reiter (Booz Allen Hamilton)
* Bokdeuk Jeong (Samsung)
* Drasko Draskovic
* Gregory Starkey (Booz Allen Hamilton)
* Henk Birkholz
* Jorg Rodel (Suse)
* Manu Foutaine (Hushmesh)
* Matthieu Legre (Cysec)
* Nathaniel McCallium (Red Hat)
* Nick Vidal

> * Nicklas Maeding (IBM)

Reviewer 2: Nicolas , please :)

Author: Updated in #235

* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

#### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* Arm - Nathaniel McCallum +  / Michael Lu
* Google - Cfir Cohen  / Catalin Sandu 
* Huawei - Zhipeng (Howard) Huang 
* Intel - Dan Middleton (TAC chair) + / Simon Johnson
* Meta - Eric Northup / Shankaran Gnanashanmugam /
* Microsoft - Alec Fernandez +
* Nvidia - Fritz Alder +
* Red Hat - Lily Sturmann  / Yash Mankad +
* TikTok - Mingshen Sun + / Yao Zhang

   " + " indicates attending

#### Other Attendees

* Bob Reiter (Booz Allen Hamilton)
* Bokdeuk Jeong (Samsung)
* Drasko Draskovic
* Gregory Starkey (Booz Allen Hamilton)

> * Henry Wang (Meta)

Reviewer 3: moved to above
```suggestion
```

* Henk Birkholz
* Jorg Rodel (Suse)

> * Kevin Hui (Meta)

Reviewer 3: moved to above
```suggestion
```

> *  Kevin Hui (Meta)

Reviewer 3: ```suggestion
* Kevin Hui (Meta)
```
typo / whitespace

* Manu Foutaine (Hushmesh)
* Matthieu Legre (Cysec)
* Nathaniel McCallium (Red Hat)
* Nick Vidal

> * Nicklas Maeding

Author: Updated, thank you.

> * Pawan Khandavilli

Reviewer 3: ```suggestion
* Pawan Khandavilli (Microsoft)
```

* Rachel Wan (IBM)

> * Richard Zak

Reviewer 3: ```suggestion
* Richard Zak (Enarx Maintainer; Booz Allen Hamilton)
```

* Rachel Wan from IBM tried to introduce themselves via chat.
* Bob Reiter from Booz Allen introduced themselves.
* Gregory Starkey introduced themselves.
* Kevin Hui from Meta introduced themselves and expressed their interest in attestation.
* Nicolas Maeding from IBM introduced themselves.

## Old Business

DM reviewed highlights from the last TAC meeting. DM then called out the need for comments and feedback on the Payload Governance Patterns and Mentorship proposals.

## Announcements

Fritz Alder stated that he is the new voting member from Nvidia and that he would be regularly attending the TAC meetings.
  

## Exec Director Update

Mike Bursell was unable to attend and therefore there was not Executive Director update. 

## New Outreach Sig Demos

Pawan Khandavilli (PK) from Microsoft briefly introduced themselves. PK stated that the Outreach team identified demos as a missing part of their events and marketing strategy. PK stated that the team is collecting demos and suggested that videos might be the best approach. PK asked the TAC to submit demos to him and Outreach so they can be potentially showcased at future events. 

## Enarx Annual Review

> Richard Zak (RZ), Gregory Starkey (GS), and Nick Vidal (NV) presented the [Enarx annual review](./VirTee%20Demo.pdf). Discussion ensued.

Reviewer 4: ```suggestion
Richard Zak (RZ), Gregory Starkey (GS), and Nick Vidal (NV) presented the [Enarx annual review](./Enarx%20Review%202024.pdf). Discussion ensued.
```

Author: Good catch, thank you.
